content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_design_dialog/.content.xml

```diff
@@ -114,6 +114,22 @@
                             </appResourcesClientlib>
                         </items>
                     </properties>
+                    <analyticssection
```

Comment:
I'd make this part of the `properties` tab.

Reply:
fetching from properties tab under page policy now